src/implementations/vanilla/components/menu/_menu.scss

```diff
@@ -7,23 +7,89 @@
 @use '@ecl/theme-dev/theme';
 @use '@ecl/vanilla-layout-grid/mixins/breakpoints';
 
+$_main-nav-background: null !default;
+$_main-nav-shadow: null !default;
+$_menu-items-distribution: null !default;
+$_menu-desktop-padding-top: null !default;
+$_menu-desktop-vertical-padding: null !default;
+$_menu-dropdown-background: null !default;
+$_menu-dropdown-border-color: null !default;
+$_menu-dropdown-item-hover-color: null !default;
+$_menu-dropdown-shadow: null !default;
+$_menu-expanded-background: null !default;
+$_menu-header-horizontal-padding: null !default;
+$_menu-item-border: null !default;
+$_menu-item-weight: null !default;
+$_menu-item-hover-background: null !default;
+$_menu-item-hover-border: null !default;
+$_menu-item-hover-color: null !default;
+$_menu-item-icon-margin: null !default;
+$_menu-link-active-background: null !default;
+$_menu-link-active-border-width: null !default;
+$_menu-link-active-border-color: null !default;
+$_menu-link-border-radius: null !default;
+$_menu-link-border-width: null !default;
+$_menu-link-border-focus-color: null !default;
+$_menu-link-desktop-height: null !default;
+$_menu-link-hover-background: null !default;
+$_menu-link-focus-background: null !default;
+$_menu-link-focus-color: null !default;
+$_menu-link-mobile-padding: null !default;
+$_menu-link-no-arrow-color: null !default;
+$_menu-link-outline-color: null !default;
+$_menu-link-padding-right: null !default;
+$_menu-list-background: null !default;
+$_menu-mega-padding-top: null !default;
+$_menu-mega-padding-bottom: null !default;
+$_menu-mobile-active-background: null !default;
+$_menu-mobile-background: null !default;
+$_menu-mobile-header-background: null !default;
+$_menu-mobile-header-shadow: null !default;
+$_menu-mobile-shadow: null !default;
+$_menu-mobile-vertical-padding: null !default;
+$_menu-open-button-padding: null !default;
+$_menu-open-button-shadow: null !default;
+$_menu-open-outline-color: null !default;
+$_menu-open-button-radius: null !default;
+$_menu-open-button-border: null !default;
+$_menu-open-focus-border: null !default;
+$_menu-open-vertical-margin: null !default;
+$_menu-overlay-background: null !default;
+$_menu-subitem-border: null !default;
+$_menu-subitem-background: null !default;
+$_menu-subitem-margin-bottom: null !default;
+$_menu-subitem-margin-top: null !default;
+$_menu-sublink-desktop-margin: null !default;
+$_menu-sublink-desktop-padding: null !default;
+$_menu-sublink-mobile-vertical-padding: null !default;
+$_menu-sublink-outline-offset: null !default;
+$_menu-sublink-active-color: null !default;
+$_menu-sublink-active-weight: null !default;
+$_menu-sublink-active-background: null !default;
+$_menu-sublink-hover-color: null !default;
+$_menu-title-font-weight: null !default;
+$_mobile-item-color: null !default;
+$_mobile-item-hover-background: null !default;
```

Comment:
wow, that's a lot of variables!
that's a good example of "small differences" between EC and EU going out of control